### PR TITLE
bump csso version

### PR DESCRIPTION
**Pull request by lahmatiy**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -20,7 +20,9 @@ CssoFilter.prototype.extensions = ['css'];
 CssoFilter.prototype.targetExtension = 'css';
 
 CssoFilter.prototype.processString = function (str) {
-	return csso.justDoIt(str, this.options.restructure);
+	return csso.minify(str, {
+		restructuring: this.options.restructure
```
Comment (sindresorhus) [inline index.js]: Just pass `this.options` directly to `css.minify` so it's forward compatible for additional options.

Comment (lahmatiy) [inline index.js]: This change will break backward capability. Is it ok?

Comment (sindresorhus) [inline index.js]: I won't and yes.

Comment (lahmatiy) [inline index.js]: Done

**Pull request merged**